code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -344,6 +344,11 @@
 	if(C.stat == DEAD || (C.status_flags & FAKEDEATH))
 		return 0 //welp too late for them!
 
+	for(var/X in C.bodyparts)
+		var/obj/item/bodypart/part = X
+		if(part.status != BODYPART_ORGANIC)
+			return 0 // No flesh to inject into.

```

Comment:
This will return a `0` on the first robotic bodypart it finds. You could put this whole proc at the end and return a `1` on a single organic part being found and a `0` otherwise.

Reply:
Does `for` scan all applicable objects? Because I tried to robust a half augged human, and it wouldn't heal them.
🤔

Reply:
@FlattestGuitar So here's the issue. If you put a "1" in this proc, it forces a heal. So I can't quite reverse it, because then this happens when it encounters.. literally anything. Injured or not.

![dreamseeker_2018-02-01_18-46-46](https://user-images.githubusercontent.com/26494679/35712673-76b4d394-0780-11e8-987c-03b7c8c7167e.png)

Constantly Forever,